apps/web/app/(authenticated)/(app)/app/desktop-sidebar.tsx

```diff
@@ -68,16 +72,23 @@ export const DesktopSidebar: React.FC<Props> = ({ workspace, className }) => {
             <h3 className="text-xs font-semibold leading-6 text-content">Your APIs</h3>
             <ul className="mt-2 -mx-2 space-y-1">
               {workspace.apis.map((api) => (
-                <li key={api.id}>
-                  <NavLink
-                    item={{
-                      icon: Code,
-                      href: `/app/apis/${api.id}`,
-                      label: api.name,
-                      active: segments.includes(api.id),
-                    }}
-                  />
-                </li>
+                <Tooltip>
+                  <TooltipTrigger className="w-full overflow-hidden text-ellipsis">
+                    <li key={api.id}>
```

Comment:
This `<li>` with the key should remain the most outer thing, the key is used by react to deduplicate rendering